README.md

```diff
@@ -1 +1,46 @@
-# g2p_seq2seq_pytorch
\ No newline at end of file
+# G2P for PyTorch
+
+This repo implements a G2P model and APIs using the PyTorch framework. The two APIs expressed here are:
+```
+- load_g2p_model
+- decode_word
+```
+
+The APIs also optionally expose phoneme stress information and beam search.
+
+## Download the model
+
+Download the model from this [link](https://cisco-my.sharepoint.com/:u:/p/vijayrk/EZWq-McCQdBOmu7GrGojWGwB1jsKfeN9xNICsaJmH4WrGg?e=oXhw22)
```

Comment:
Hi Vijay, here's a DevNet link you can use: https://developer.cisco.com/fileMedia/download/5b20821d-f092-3b57-a438-546046ffaa61 Please test to make sure it works as you expect and let me know.

Reply:
Thanks so much!